src/sentry/middleware/integrations/parsers/jira.py

```diff
@@ -73,7 +74,11 @@ def get_response(self):
             )
 
         if self.view_class in self.immediate_response_region_classes:
-            return self.get_response_from_region_silo(region=regions[0])
+            try:
+                return self.get_response_from_region_silo(region=regions[0])
+            except ApiError as err:
```

Comment:
What kind of error are expecting here that getting response from control would fix it?

Reply:
We get an `ApiRetryError` if all retries are exhausted.